marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/deploy/DeployHubAmpsCommand.java

```diff
@@ -18,6 +18,15 @@
  */
 public class DeployHubAmpsCommand extends DeployAmpsCommand {
 
+    /**
+     * DeployHubAmpsCommand should run before we run LoadHubModulesCommand as we require the amp for getting the default
+     * rewriter to be present so that it can be used to generate the custom rewriter.
+     */
+   public DeployHubAmpsCommand() {
+       super();
+       setExecuteSortOrder(390);
```

Comment:
Rob filed https://github.com/marklogic-community/ml-app-deployer/issues/432 . We may not need this if there is a new release.

Reply:
Yeah, I did talk to @rjrudin about this. This is an interim fix for now.